epub-a11y-meta-guide/1.0/draft/index.html

```diff
@@ -1379,58 +1379,614 @@ <h5>Sign Language</h5>
+                <section id="metadata-a11y-features-ChemML">
+                    <h5>ChemML</h5>
+
+                    <p>Indicates that chemical notations, such as molecular formulas and reaction diagrams, are represented using the Chemical Markup Language (ChemML, sometimes also refered to as CML). This provides a structured, accessible alternative to presenting chemical information as flat images.</p>
+                    <p>Using ChemML allows assistive technologies to correctly interpret and announce chemical structures. It also enables searching, copying, and styling of the notation, making it highly adaptable to user needs.</p>
+
+                    <aside class="example" title="AccessibilityFeature: ChemML">
+                        <p>Instead of an inaccessible image of a water molecule, the structure is encoded directly in the content document using ChemML. This allows a reading system or assistive technology to identify the atoms and their bonds.</p>
+
+                        <p><strong>Example of H₂O represented in ChemML:</strong></p>
+                        <pre>
+                            <code>&lt;!DOCTYPE html&gt;
+                &lt;html xmlns="http://www.w3.org/1999/xhtml"&gt;
+                &lt;head&gt;
+                  &lt;title&gt;Chemistry Example&lt;/title&gt;
+                &lt;/head&gt;
+                &lt;body&gt;
+                  &lt;p&gt;The structure of a water molecule (H₂O) is shown below:&lt;/p&gt;
+
+                  &lt;!-- Chemical Markup Language provides a structured format for chemistry --&gt;
+                  &lt;cml xmlns="http://www.xml-cml.org/schema"&gt;
+                    &lt;molecule id="water"&gt;
+                      &lt;atomArray&gt;
+                        &lt;atom id="a1" elementType="O" /&gt;
+                        &lt;atom id="a2"elementType="H" /&gt;
+                        &lt;atom id="a3" elementType="H" /&gt;
+                      &lt;/atomArray&gt;
+                      &lt;bondArray&gt;
+                        &lt;bond atomRefs2="a1 a2" order="S" /&gt;
+                        &lt;bond atomRefs2="a1 a3" order="S" /&gt;
+                      &lt;/bondArray&gt;
+                    &lt;/molecule&gt;
+                  &lt;/cml&gt;
+
+                &lt;/body&gt;
+                &lt;/html&gt;</code>
+                        </pre>
 
-					</section>
+                        <p>The metadata to include in the package document to advertise this feature would be:</p>
+                        <pre>
+                            <code>&lt;meta property="schema:accessibilityFeature"&gt;ChemML&lt;/meta&gt;</code>
+                        </pre>
+                    </aside>
+                </section>
 
-					<section id="metadata-a11y-features-latex-chemistry">
-						<h5>Chemistry Represented using LaTeX</h5>
+               <section id="metadata-a11y-features-latex-chemistry">
+                    <h5>Chemistry Represented using LaTeX</h5>
```

Comment:
This always makes me wonder if anyone puts raw latex into epubs. I see evidence of people using it to generate epubs, but nothing about putting it in raw. Do you get content like this in Bookshare?